File: GotravelDesktop/src/main/java/com/gotravel/Model/Pago.java
package com.gotravel.Model;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Pago {

    private int id;
    private Double coste;
    private String fecha;

}

File: GoTravelMobile/app/src/main/java/com/gotravel/mobile/data/model/Pago.kt
package com.gotravel.mobile.data.model

import kotlinx.serialization.Serializable
import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Serializable
data class Pago (
    val id: Int? = null,
    val coste: Double,
    val fecha: String
)